autoload/lsc/edit.vim

```diff
@@ -142,6 +142,7 @@ function! s:ApplyAll(changes) abort
     else
       let l:cmd .= ' edit '.l:file_path
     endif
+    let l:cmd .= ' | silent execute "set fdl=99"'
```

Comment:
This won't let use restore folds after we are done. If instead we use `"set nofoldenable"` we could later do `"set foldenable"` to restore them.